examples/bpk-component-ticket/examples.js

```diff
@@ -141,28 +141,6 @@ const WithoutPaddingImageExample = () => (
   </BpkTicket>
 );
 
-const WithoutNotchesExample = () => (
```

Comment:
Please add one more example for `padded=false` and `vertical=true`

Reply:
Have added. Thanks